root/static/scripts/edit/URLCleanup.js

```diff
@@ -821,6 +835,15 @@ validationRules[LINK_TYPES.bbcmusic.artist] = function (url) {
   return /bbc\.co\.uk\/music\/artists\//.test(url);
 };
 
+// allow only BookBrainz entity pages with the BookBrainz rel
+function validateBookBrainz(url) {
+  return /bookbrainz\.org\/[^\/]*\/[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}/.test(url);
```

Comment:
Neither do I. More info for later on: http://chatlogs.metabrainz.org/brainzbot/metabrainz/msg/3618257/